pkg/odo/cli/url/create.go

```diff
@@ -110,6 +119,7 @@ func NewCmdURLCreate(name, fullName string) *cobra.Command {
 	}
 	urlCreateCmd.Flags().IntVarP(&o.urlPort, "port", "", -1, "port number for the url of the component, required in case of components which expose more than one service port")
 	urlCreateCmd.Flags().BoolVar(&o.urlOpenFlag, "open", false, "open the created link with your default browser")
+	urlCreateCmd.Flags().StringVarP(&o.outputFlag, "output", "o", "", "gives output in the form of json")
```

Comment:
no true. This specifies the output format. Help message should also mention what formats are valid

Reply:
It would be good to abstract this away to make sure that we have the same message for this flag everywhere